src/main/java/io/github/thebusybiscuit/slimefun4/api/items/SlimefunItem.java

```diff
@@ -70,6 +74,14 @@
  */
 public class SlimefunItem implements Placeable {
 
+    /**
+     * This is the set of allowed wiki domains.
+     */
+    private static final Set<String> ALLOWED_WIKI_DOMAIN = Set.of(
+        "github.com",
+        "github.io"
+    );
```

Comment:
That's just really nonsensical in my opinion, first off: Anything can be put onto a `github.io` page, so that supposed "security" isn't much.

Second of all, any file can be uploaded to github and linked to this way too.

So I really don't see a point in this filter at all.